src/components/OverviewPage/TransactionList.tsx

```diff
@@ -46,69 +35,29 @@ const OverviewPageTransactionList = ({ className, onTransactionClick }: Overview
   const { addresses, fetchAddressTransactionsNextPage, isLoadingData } = useAddressesContext()
   const totalNumberOfTransactions = addresses.map((address) => address.details.txNumber).reduce((a, b) => a + b, 0)
 
-  const allConfirmedTxs = addresses
-    .map((address) => address.transactions.confirmed.map((tx) => ({ ...tx, address })))
-    .flat()
-    .sort((a, b) => b.timestamp - a.timestamp)
-
-  const allPendingTxs = addresses
-    .map((address) => address.transactions.pending.map((tx) => ({ ...tx, address })))
-    .flat()
-    .sort((a, b) => b.timestamp - a.timestamp)
+  const allConfirmedTxs = getTransactionsForAddresses('confirmed', addresses)
+  const allPendingTxs = getTransactionsForAddresses('pending', addresses)
 
   const loadNextTransactionsPage = async () => {
     addresses.forEach((address) => fetchAddressTransactionsNextPage(address))
   }
 
   const showSkeletonLoading = isLoadingData && !allConfirmedTxs.length && !allPendingTxs.length
 
-  const transactionsTableHeadersI18n = transactionsTableHeaders.map((el) => ({ ...el, title: t(el.title) }))
-
   return (
-    <Table headers={transactionsTableHeadersI18n} isLoading={showSkeletonLoading} className={className}>
+    <Table isLoading={showSkeletonLoading} className={className} minWidth="500px">
       {allPendingTxs
         .slice(0)
         .reverse()
-        .map(({ txId, timestamp, address, amount, type }) => (
-          <TableRow key={txId} columnWidths={tableColumnWidths} blinking>
-            <TableCell>
-              <TransactionalInfo content={t`Pending`} type="pending" />
-            </TableCell>
-            <TableCell>{dayjs(timestamp).fromNow()}</TableCell>
-            <TableCell>
-              <AddressBadge address={address} />
-            </TableCell>
-            <TableCell align="end">
-              {type === 'transfer' && amount && <TransactionalInfo type="out" prefix="-" content={amount} amount />}
-            </TableCell>
+        .map(({ data: tx, address }: BelongingToAddress<SimpleTx>) => (
+          <TableRow key={tx.txId} blinking>
+            {tx.type === 'transfer' && <TransactionalInfo transaction={tx} addressHash={address.hash} />}
           </TableRow>
         ))}
-      {allConfirmedTxs.map((transaction) => {
-        const amount = calAmountDelta(transaction, transaction.address.hash)
-        const amountIsBigInt = typeof amount === 'bigint'
-        const isOut = amountIsBigInt && amount < 0
-
+      {allConfirmedTxs.map(({ data: tx, address }: BelongingToAddress<Transaction>) => {
         return (
-          <TableRow
-            key={`${transaction.hash}-${transaction.address.hash}`}
-            columnWidths={tableColumnWidths}
-            onClick={() => onTransactionClick(transaction)}
-          >
-            <TableCell>
-              <TransactionalInfo content={isOut ? '↑ ' + t`Sent` : '↓ ' + t`Received`} type={isOut ? 'out' : 'in'} />
```

Comment:
Remove both from App.json

Reply:
These are still used in the transaction details modal

Reply:
Aren't they with lower case there? I think it's important to distringuish.